app/controllers/projects_controller.rb

```diff
@@ -1,9 +1,16 @@
 class ProjectsController < ApplicationController
+  include JSONAPI::Deserialization
```

Comment:
This isn't necessary for show routes. This is a helper for deserializing JSONAPI-formatted request bodies and is used to unpack update attributes in the TranscriptionController, but that's not used in these new show methods.

```suggestion
```